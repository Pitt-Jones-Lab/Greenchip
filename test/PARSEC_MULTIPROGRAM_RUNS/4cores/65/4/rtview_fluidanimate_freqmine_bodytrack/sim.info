{'benchmarks_revision': ('cat: .gitid: No such file or directory',),
 'benchmarks_root': ('/homes/grads/dek61/SNIPER/benchmarks',),
 'cmdline': ['/homes/grads/dek61/SNIPER/sniper/run-sniper',
             '-n',
             '4',
             '-m',
             'localhost',
             '-d',
             '/scratch/nbp3/PARSEC_MULTIPROGRAM_SHELL/4cores/65/4/rtview_fluidanimate_freqmine_bodytrack',
             '-c',
             '/homes/grads/nbp3/SNIPER_CONFIGS/gainestown_65_4_4.cfg',
             '-s',
             'stop-by-icount:10000000000',
             '--curdir=/scratch/nbp3/PARSEC_MULTIPROGRAM_SHELL/4cores/65/4/rtview_fluidanimate_freqmine_bodytrack',
             '--trace-manual',
             '-g',
             '--traceinput/enabled=true',
             '-g',
             '--traceinput/emulate_syscalls=true',
             '-g',
             '--traceinput/num_apps=4',
             '-g',
             '--traceinput/trace_prefix=/tmp/tmpcurzt8/run_benchmarks'],
 'git_revision': '68418dbe7cab15c65f1f4d300da0c505329beedd',
 'host': 'sb18',
 'pin_version': 'unknown',
 'rusage': (23613.744165,
            841.409086,
            205760,
            0,
            0,
            0,
            48958,
            0,
            0,
            8,
            1368,
            0,
            0,
            0,
            54170192,
            4034377),
 'snipercmd': ['bash',
               '-c',
               '/homes/grads/dek61/SNIPER/sniper/lib/sniper -c /homes/grads/dek61/SNIPER/sniper/config/base.cfg --general/total_cores=4 --general/output_dir=/scratch/nbp3/PARSEC_MULTIPROGRAM_SHELL/4cores/65/4/rtview_fluidanimate_freqmine_bodytrack --config=/homes/grads/dek61/SNIPER/sniper/config/nehalem65.cfg --config=/homes/grads/nbp3/SNIPER_CONFIGS/gainestown_65_4_4.cfg --traceinput/enabled=true --traceinput/emulate_syscalls=true --traceinput/num_apps=4 --traceinput/trace_prefix=/tmp/tmpcurzt8/run_benchmarks -g --traceinput/stop_with_first_app=true -g --traceinput/restart_apps=false -g --hooks/numscripts=1 -g --hooks/script0name=/scratch/nbp3/PARSEC_MULTIPROGRAM_SHELL/4cores/65/4/rtview_fluidanimate_freqmine_bodytrack/sim.scripts.py -g --hooks/script0args='],
 't_elapsed': 11294.321422100067,
 't_start': 1463345068.396096,
 'tracecmds': [],
 'user': 'nbp3',
 'vmem': 1555443712L}
