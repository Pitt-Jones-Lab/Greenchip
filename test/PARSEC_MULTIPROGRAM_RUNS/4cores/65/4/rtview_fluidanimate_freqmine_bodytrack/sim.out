                                   | Core 0      | Core 1      | Core 2      | Core 3     
  Instructions                     |  2922473393 |  2709952528 |  2678112661 |  1688380410
  Cycles                           |  2425836742 |  2425836742 |  2425836742 |  2425836742
  IPC                              |        1.20 |        1.12 |        1.10 |        0.70
  Time (ns)                        |   911968700 |   911968700 |   911968700 |   911968700
  Idle time (ns)                   |       95338 |   173401906 |      329452 |   294004537
  Idle time (%)                    |        0.0% |       19.0% |        0.0% |       32.2%
Branch predictor stats             |             |             |             |            
  num correct                      |   354546443 |   318884616 |   374702714 |   169157933
  num incorrect                    |    28493210 |    21143117 |    27160875 |    20635068
  misprediction rate               |       7.44% |       6.22% |       6.76% |      10.87%
  mpki                             |        9.75 |        7.80 |       10.14 |       12.22
TLB Summary                        |             |             |             |            
  I-TLB                            |             |             |             |            
    num accesses                   |   404418167 |   351105853 |   341327448 |   193194286
    num misses                     |        4272 |         209 |         187 |         381
    miss rate                      |       0.00% |       0.00% |       0.00% |       0.00%
    mpki                           |        0.00 |        0.00 |        0.00 |        0.00
  D-TLB                            |             |             |             |            
    num accesses                   |  1038774442 |   946811504 |  1122615379 |   512905700
    num misses                     |      468105 |     1179265 |      683895 |      512306
    miss rate                      |       0.05% |       0.12% |       0.06% |       0.10%
    mpki                           |        0.16 |        0.44 |        0.26 |        0.30
  L2 TLB                           |             |             |             |            
    num accesses                   |      472377 |     1179474 |      684082 |      512687
    num misses                     |      128022 |      336773 |      146146 |      125280
    miss rate                      |      27.10% |      28.55% |      21.36% |      24.44%
    mpki                           |        0.04 |        0.12 |        0.05 |        0.07
Cache Summary                      |             |             |             |            
  Cache L1-I                       |             |             |             |            
    num cache accesses             |   404418167 |   351105853 |   341327448 |   193194286
    num cache misses               |     6253578 |      940428 |      911432 |      946385
    miss rate                      |       1.55% |       0.27% |       0.27% |       0.49%
    mpki                           |        2.14 |        0.35 |        0.34 |        0.56
  Cache L1-D                       |             |             |             |            
    num cache accesses             |  1038774442 |   946811504 |  1122615379 |   512905700
    num cache misses               |     4945450 |     6338526 |     6530685 |     5179537
    miss rate                      |       0.48% |       0.67% |       0.58% |       1.01%
    mpki                           |        1.69 |        2.34 |        2.44 |        3.07
  Cache L2                         |             |             |             |            
    num cache accesses             |    11440924 |     7493319 |     7750739 |     6336191
    num cache misses               |     1455318 |     1989258 |     1608546 |     1199340
    miss rate                      |      12.72% |      26.55% |      20.75% |      18.93%
    mpki                           |        0.50 |        0.73 |        0.60 |        0.71
DRAM summary                       |             |             |             |            
  num dram accesses                |     9954330 |           0 |           0 |           0
  average dram access latency (ns) |      134.43 |         inf |         inf |         inf
  average dram queueing delay      |      134.43 |        0.00 |        0.00 |        0.00
Coherency Traffic                  |             |             |             |            
  num loads from dram              |      942143 |      941127 |     1086470 |      777108
  num loads from dram cache        |           0 |           0 |           0 |           0
  num loads from remote cache      |           0 |           0 |           0 |           0
